### PR TITLE
docs: add settings and caps, enforceAppInstall, activeAppDetectionPoint, defaultActiveApplication and reduceMotion

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Relevant changes have not been published yet.
Maybe, they will be the next release of 1.15.0 (Won't merge yet)

enforceAppInstall : https://github.com/appium/appium-android-driver/pull/573
screenPoint : https://github.com/appium/WebDriverAgent/pull/215
defaultActiveApplication: https://github.com/appium/WebDriverAgent/pull/209
reduceMotion : https://github.com/appium/appium-xcuitest-driver/pull/1065

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/advanced-concepts/settings.md [hunk 1]
```diff
@@ -80,3 +80,6 @@ Note that the actual commands you would use in your test script differ based on
 |`mjpegScalingFactor`| Changes the scale of screenshots. Defaults to `100`, no scaling. Integer between `1` and `100` are available. | e.g. `1`, `50`, `100` |1.12.0+|
 |`keyboardAutocorrection`| Changes the 'Auto-Correction' preference in _Keyboards_ setting. Defaults to `false` when WDA starts as xctest. | `true`, `false` |1.14.0+|
 |`keyboardPrediction`| Changes the 'Predictive' preference in _Keyboards_ setting. Defaults to `false` when WDA starts as xctest. | `true`, `false` |1.14.0+|
+|`reduceMotion`| Changes the 'reduce motion' preference of accessibility feature. | `true`, `false` |1.16.0+?|
+|`defaultActiveApplication`| Set the default active application explicitly. It helps to handle active applications in split screen, for example. Defaults to `auto` which handles on screen application as active application. | e.g., `com.apple.Preferences` |1.16.0+?|
+|`screenPoint`| Change the current screen point to get the element on current screen. It affects getting current active application. You can change this point when you test split screen, for example. The format is comma separate like `x,y` as string. | e.g. `100,300` |1.16.0+?|
```
[the remark this answers is not in the document]
Comment (jlipps) [reply docs/en/advanced-concepts/settings.md]: is `screenPoint` a little generic? should it be something like `activeAppDetectionPoint`?

[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply docs/en/advanced-concepts/settings.md]: Sorry @jlipps , I'm not very inventive in such things. %)
Of course you can create a PR and give it a better name.

[the remark this answers is not in the document]
Comment (jlipps) [reply docs/en/advanced-concepts/settings.md]: oh right, this is just the docs PR so the other one has already been merged.

[the remark this answers is not in the document]
Comment (KazuCocoa) [reply docs/en/advanced-concepts/settings.md]: 📝 
The latest WDA has `activeAppDetectionPoint`

**Pull request merged**